Tasks/VsTest/distributedtest.ts

```diff
@@ -81,6 +80,10 @@ export class DistributedTest {
             utils.Helper.addToProcessEnvVars(envVars, 'DTA.ForcePlatformV2', 'true');
         }
 
+        if (utils.Helper.isDebugEnabled()) {
+            utils.Helper.addToProcessEnvVars(envVars, 'DTA.EnableDiagLogs', 'true');
```

Comment:
I hope we are handling on dtaexecutionhost side when this variable is not set

Reply:
yes we check for isnullorempty